hexa/ui/templates/ui/icons/organisation.html

```diff
@@ -0,0 +1,11 @@
+<svg
```

Comment:
Where does this icon come from? If it's from heroicons, we should keep the "heroicons" name.

Reply:
Random SVG found on internet without license + manipulation in inkscape.